pymeasure/experiment/results.py

```diff
@@ -151,20 +151,35 @@ class Results(object):
 
     :cvar COMMENT: The character used to identify a comment (default: #)
     :cvar DELIMITER: The character used to delimit the data (default: ,)
-    :cvar LINE_BREAK: The character used for line breaks (default \\n)
+    :cvar LINE_BREAK: The character used for line breaks (default: \\n)
     :cvar CHUNK_SIZE: The length of the data chuck that is read
 
     :param procedure: Procedure object
     :param data_filename: The data filename where the data is or should be
                           stored
+    :param rollover: When to rollover the results files.
+                     If None, no rollover occurs.
+                     If 'size', rollover occurs once the current file size
+                     exceeds a given size.
+                     See logging.handlers.RotatingFilehandler.
+                     If 'time', rollover occurs at a speificed time or time interval.
+                     See logging.handlers.TimedRotatingFileHandler.
```

Comment:
Might be convenient if you link logging.handlers.... to their respective documentation pages. 
You can do this by changing e.g. the following line:
```suggestion
                     See `logging.handlers.TimedRotatingFileHandler`_.
```
and adding the following to the end of the docstring: 
```
    .. _logging.handlers.TimedRotatingFileHandler: https://docs.python.org/3/library/logging.handlers.html#timedrotatingfilehandler
```